examples/airflow_orchestration/run.py

```diff
@@ -11,13 +11,33 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
+import os
 import random
+from typing import Any, Type
 
+from zenml.io import fileio
+from zenml.materializers.base_materializer import BaseMaterializer
 from zenml.pipelines import pipeline
 from zenml.steps import Output, step
 
 
-@step
+class MyIntMaterializer(BaseMaterializer):
```

Comment:
why do we need this? feels like this is a change you made locally for testing that shouldn't be merged?